package.json

```diff
@@ -17,6 +17,7 @@
     "eslint": "^1.7.2",
     "istanbul": "^0.4.0",
     "mocha": "^2.3.3",
+    "chai": "^3.5.0",
```

Comment:
I'd strongly discourage adding a big library as dependency just for one simple function. See comment below